client/infrastructure/electron/app_paths.ts

```diff
@@ -43,10 +43,22 @@ export function getAppPath() {
   return electronAppPath;
 }
 
-export function pathToEmbeddedBinary(filename: string) {
-  let osName = os.platform().toString(); /* should be either 'linux' or 'win32' */
-  if (osName === 'win32') {
-    osName = 'windows';
+export function pathToEmbeddedTun2socksBinary() {
+  return path.join(
+    unpackedAppPath(), 'client', 'output', 'build',
```

Comment:
Ugh, I guess it was already like that, but this is a very bad path. This is inside the app, why does it have `client/output/build/os`? That can lead to many issues. For example, like we could be packaging extra build artifacts that shouldn't be there (e.g. adding the linux stuff to windows). It's also confusing.

Can we make it so that the binary is in `unpackedAppPath(), "bin", "tun2socks[.exe]"`?

Reply:
Ideally yes, please check my replies about the runtime paths to @daniellacosse 's electron path refactoring email. P.S. we won't include any Linux bits in Windows (or Windows bits in Linux), the platform folder is basically useless. Actually the entire `client/output/build/<platform>` is useless, there are no other files in these folders.